db-cleanup/airflow-db-cleanup.py

```diff
@@ -1,10 +1,15 @@
 from airflow.models import DAG, DagRun, TaskInstance, Log, XCom, SlaMiss, DagModel, Variable
 from airflow.jobs import BaseJob
 from airflow.models import settings
-from airflow.operators import PythonOperator
+from airflow.operators.python_operator import PythonOperator
 from datetime import datetime, timedelta
 import os
 import logging
+try:
+    from airflow.utils import timezone
+    now = timezone.utcnow
+except ImportError:
+    now = datetime.utcnow
```

Comment:
Whats the difference between these XXX.utcnow outputs? Can we use just one?

Reply:
For 1.7 and 1.8 it will fall back to Python default datetime.utcnow. For 1.9 and above it will use timezone.utcnow from airflow.utils